.travis.yml

```diff
@@ -14,7 +14,7 @@ script:
     - julia -e 'Pkg.add("Images")'  # Needed for Docs 
     - julia -e 'Pkg.clone("https://github.com/JuliaImages/ImageDraw.jl")' # Needed for Docs 
     - julia -e 'Pkg.clone(pwd()); Pkg.build("ImageFeatures")'
-    - julia -e 'Pkg.test("ImageFeatures", coverage=true)'
+    - julia -e 'Pkg.test("ImageFeatures",coverage=true)'
```

Comment:
Yes, they run quite fast with coverage off, but tend to take >45 mins (which is maximum time allowed by travis per job) if the coverage is switched on.